controllers/internal/csiscaleoperator/csiscaleoperator.go

```diff
@@ -33,11 +33,11 @@ func (c *CSIScaleOperator) Unwrap() *csiv1.CSIScaleOperator {
 //func (c *CSIScaleOperator) GetLabels() labels.Set {
 func (c *CSIScaleOperator) GetLabels() map[string]string {
 	labels := labels.Set{
-		"app.kubernetes.io/name":       config.SALabelName,
-		"app.kubernetes.io/instance":   config.SALabelInstance,
-		"app.kubernetes.io/managed-by": config.SALabelManage,
-		"product":                      config.Product,
-		"release":                      config.SALabelRelease,
+		config.LabelAppName:      config.ResourceAppName,
+		config.LabelAppInstance:  config.ResourceInstance,
+		config.LabelAppManagedBy: config.ResourceManagedBy,
+		config.LabelProduct:      config.Product,
+		config.LabelRelease:      config.Release,
```

Comment:
Here, for config.LabelRelease the value should be "ibm-spectrum-scale-csi-operator".
In Ansible operator:
csiscaleoperator/defaults/main.yml:labelRelease: "ibm-spectrum-scale-csi-operator"
csiscaleoperator/templates/sa-attacher.yaml.j2:    release: '{{ labelRelease }}'

Reply:
Changed the value to `ibm-spectrum-scale-csi-operator` as in Ansible templates.